eng/Version.Details.xml

```diff
@@ -5,14 +5,18 @@
       <Uri>https://github.com/dotnet/windowsdesktop</Uri>
       <Sha>c0d88e97c24130c78429dbee4444e4941b1c26f6</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.WindowsDesktop.App" Version="6.0.0-alpha.1.20559.4" CoherentParentDependency="Microsoft.NET.Sdk">
+    <Dependency Name="VS.Redist.Common.WindowsDesktop.SharedFramework.x64.6.0" Version="6.0.0-alpha.1.20559.4" CoherentParentDependency="Microsoft.NET.Sdk">
```

Comment:
The added dependencies need to be added to dotnet/sdk for coherency purposes, if they do not already exist.

Reply:
I'll open a PR to add the new dependencies to dotnet/sdk.

Reply:
dotnet/sdk pr open at https://github.com/dotnet/sdk/pull/14512